web-components/src/components/form/CreditTransferForm.tsx

```diff
@@ -0,0 +1,263 @@
+import TextField from '../inputs/TextField';
+import AmountField from '../inputs/AmountField';
+import Description from '../description';
+import CheckboxLabel from '../inputs/CheckboxLabel';
+import {
+  CreditRetireFields,
+  FormValues as RetireFormValues,
+  validateCreditRetire,
+  initialValues as initialValuesRetire,
+} from './CreditRetireForm';
+import Submit from './Submit';
+import {
+  requiredMessage,
+  invalidAmount,
+  insufficientCredits,
+} from '../inputs/validation';
+
+/**
+ * Send sends tradable credits from one account to another account.
+ * Sent credits can either be tradable or retired on receipt.
+ * https://docs.regen.network/modules/ecocredit/03_messages.html#msgsend
+ *
+ * Validation:
+ *    sender: must ba a valid address, and their signature must be present in the transaction
+ *    recipient: must ba a valid address
+ *    credits: must not be empty
+ *    batch_denom: must be a valid batch denomination
+ *    tradable_amount: must not be negative
+ *    retired_amount: must not be negative
+ *  if retired_amount is positive:
+ *    retirement_location: must be a valid location
+ *
+ * Also:
+ * https://docs.regen.network/modules/ecocredit/protobuf.html#msgsend
+ * https://docs.regen.network/modules/ecocredit/protobuf.html#msgsend-sendcredits
+ */
+
+const useStyles = makeStyles((theme: Theme) => ({
+  senderField: {
+    '& label': {
+      color: `${theme.palette.primary.contrastText} !important`,
+    },
+    '& .MuiInputBase-formControl': {
+      backgroundColor: theme.palette.info.light,
+      marginTop: theme.spacing(2.25),
+    },
+  },
+  textField: {
+    marginTop: theme.spacing(10.75),
+    '& .MuiInputBase-formControl': {
+      marginTop: theme.spacing(2.25),
+    },
+  },
+  description: {
+    marginBottom: theme.spacing(5),
+    [theme.breakpoints.up('sm')]: {
+      fontSize: theme.spacing(4.5),
+    },
+    [theme.breakpoints.down('sm')]: {
+      fontSize: theme.spacing(4),
+    },
+    '& a': {
+      cursor: 'pointer',
+    },
+  },
+  checkboxLabel: {
+    marginTop: theme.spacing(10.75),
+    marginBottom: theme.spacing(10.75),
+    alignItems: 'initial',
+    '& .MuiCheckbox-root': {
+      alignSelf: 'end',
+    },
+  },
+  checkboxDescription: {
+    color: theme.palette.primary.contrastText,
+    fontSize: theme.spacing(4.5),
+    fontWeight: 'bold',
+    marginLeft: theme.spacing(1),
+  },
+  groupTitle: {
+    marginTop: theme.spacing(10.75),
+    marginBottom: theme.spacing(2),
+  },
+  stateCountryGrid: {
+    [theme.breakpoints.up('sm')]: {
+      flexWrap: 'nowrap',
+    },
+  },
+  stateCountryTextField: {
+    marginTop: theme.spacing(6),
+    [theme.breakpoints.up('sm')]: {
+      '&:first-of-type': {
+        marginRight: theme.spacing(2.375),
+      },
+      '&:last-of-type': {
+        marginLeft: theme.spacing(2.375),
+      },
+    },
+  },
+  postalCodeField: {
+    marginTop: theme.spacing(8),
+    marginBottom: theme.spacing(12.5),
+  },
```

Comment:
we can get rid of these styles now that it's been refactored in the retire form component
```suggestion
```

Reply:
well seen!! thanks a lot, I missed deleting that